clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -424,21 +505,62 @@ export const CustomTextArea = ({
     />
   );
 
-  return (
-    <FormControl isInvalid={isInvalid}>
-      {label ? (
+  // When there is no label, it doesn't matter if stacked or inline
+  // since we only render the text field
+  if (!label) {
+    return (
+      <FormControl isInvalid={isInvalid}>
+        <Flex>
+          {InnerTextArea}
```

Comment:
Looks like this was already named `InnerTextArea` but in general naming jsx variables with InitCaps can cause confusion because then it looks like it should be used like a component:`<InnerTextArea />`